pyintegration/deephaven2/_jcompat.py

```diff
@@ -71,3 +73,23 @@ def j_map_to_dict(m):
         r[k] = v
 
     return r
+
+
+T = TypeVar("T")
+R = TypeVar("R")
+
+
+def j_function(func: Callable[[T], R], dtype: DType) -> jpy.JType:
+    """Constructs a Java Function<PyObject, Object> implementation from a Python callable or an object with an
+     'apply' method that accepts a single argument.
+
+    Args:
+        func (Callable): a Python callable or an object with an `apply` method that accepts a single argument
+        dtype (DType): the return type of `func`
+
+    Returns:
+        io.deephaven.integrations.python.PythonFunction instance
+    """
```

Comment:
This doc string has some inconsistencies.
1) Function<PyObject, Object> is not quoted
2) apply is quoted with `
3) apply is quoted with '

I'd pick one quotation and apply it to all areas.